packages/jaeger-ui/src/components/DeepDependencies/SidePanel/DetailsPanel.test.js

```diff
@@ -24,7 +24,18 @@ import { UnconnectedDetailsPanel as DetailsPanel } from './DetailsPanel';
 jest.mock('../../common/VerticalResizer', () => {
   // eslint-disable-next-line react/display-name
   return ({ onChange, position }) => (
-    <div data-testid="vertical-resizer" data-position={position} onClick={() => onChange(0.6)}>
+    <div
+      data-testid="vertical-resizer"
+      data-position={position}
+      onClick={() => onChange(0.6)}
+      onKeyDown={e => {
+        if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
do we actually test key-down in the unit tests? If not then why complicate the mock?